### PR TITLE
Bump version to 5.0.0-rc2

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

**Comment by talevy** [comment]
missing this, no? https://github.com/elastic/logstash/blob/5.0/logstash-core/build.gradle#L8

**Comment by jsvd** [comment]
looking at rc1 references, it seems the last two need to be manually changed.

% grep rc1 . -r | grep -v vendor
./CHANGELOG.md:## 2.0.0-rc1 (October 22, 2015)
./docs/static/releasenotes.asciidoc:* <<RC1, Logstash 5.0.0-rc1>>
./docs/static/releasenotes.asciidoc:=== Logstash 5.0.0-rc1 Release Notes
./logstash-core/build.gradle:version = '5.0.0-rc1'
./logstash-core/lib/jars.rb:require_jar('org.logstash', 'logstash-core', '5.0.0-rc1')

**Comment by suyograo** [comment]
Sigh, I always forget :(

**Comment by talevy** [comment]
@suyograo sorry :( this was fixed in 5.x and master

**Comment by jsvd** [comment]
this needs to be automated like all other version strings are..today I'll check if there's an issue to track this or create a new one.

**Comment by jsvd** [comment]
added minor comment about lock changes unrelated to rc1 -> rc2 bump in this PR

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -407,7 +411,7 @@ GEM
       logstash-filter-json
       logstash-input-generator
       logstash-output-file
-    logstash-output-elasticsearch (5.1.2-java)
+    logstash-output-elasticsearch (5.2.0-java)
```
Comment (suyograo) [inline Gemfile.jruby-1.9.lock]: Yes, it was intentional for 5.0.0

**Comment by jsvd** [comment]
LGTM

**Pull request merged**